scripts/ondrej.js

```diff
@@ -0,0 +1,5 @@
+// ajskdfjlsadjflkasjdfkljasf
+
+// muj skript pico GTFO
```

Comment:
I dont think this kind of language is appropriate in a public repo

Reply:
yes

Reply:
LGTF